extensions/todoist/src/helpers/menu-bar.ts

```diff
@@ -1,7 +1,9 @@
-import { LaunchType, launchCommand } from "@raycast/api";
+import { LaunchType, launchCommand, captureException } from "@raycast/api";
 
 export function refreshMenuBarCommand() {
-  return launchCommand({ name: "menu-bar", type: LaunchType.Background, context: { fromCommand: true } });
+  return launchCommand({ name: "menu-bar", type: LaunchType.Background, context: { fromCommand: true } }).catch(
+    captureException,
+  );
```

Comment:
`captureException` will cause errors to be thrown to the developer hub, and I'm not sure these are really meaningful, as people may not use the menu bar command as a setting. I'd recommend silently failing here 🙂

Reply:
For sure @thomaslombart . My intention was to swallow the exception but I did not want to simply `console.error`, so `captureException` felt like a better alternative 😊. I have made another revision.